R/generate_virtual_pop.R

```diff
@@ -1,11 +1,10 @@
 #' Create a virtual population
 #'
 #' This function creates a population from a distribution.
-#' Author: Tao Guo
 #'
 #'@param N The numbers of samples
 #'@param var_name the variable name that we need to create
-#'@param dist the function that we are using
+#'@param dist the distribution function for random variables (starts with 'r') that we are using
```

Comment:
Could consider adding a couple of examples? Like `rnorm`, `rexp`, etc?

Reply:
Got it!

Reply:
Done!